iset.mm

```diff
@@ -106619,28 +106737,309 @@ According to Wikipedia ("Least common multiple", 27-Aug-2020,
+      NMVUMVVDVVEGZGZVVAVUNVVGVVAGZVUNVUEUWNQHZNFZVVHVVIVUEUWOIHZCQHZVUEVURIHZB
+      QHZXHHZNVVGVVAVVIVVOJZUVOVUGVVFVVAVVPMVUJUVOVUGGZVVFGZVVAVVPVVRVVAGVVIVUE
+      UVQIHZUVSQHZVUEVUQIHZUVSQHZVUEVUSIHZUVSQHZXHHZVVOVVRVVIVVTJVVAVVRVUEUVSUV
+      QVUGVUETFUVOVVFVUEWNXIZUVOUVSTFVUGVVFUVOUVSUWGWOZXJZUVOUVQTFVUGVVFUYRXJVV
+      RBCUVOUYMVUGVVFUYNXJZUVNUYOUVMVUGVVFUYPXKZVVRBUVMUVNVUGVVFXNWPZVVRCUVMUVN
+      VUGVVFXOWPZXPZUVOUVQPXQLVUGVVFUYSXJXROVVAVVRVVTVUEVUTIHZUVSQHZVWEVVAVVSVW
+      NUVSQUVQVUTVUEIXSWTVVRVWOVWAVWCXHHZUVSQHVWEVVRVWNVWPUVSQVVRVUEVUQVUSVWFVV
+      RBUWOVWIVVDUWOTFVVQVVEUWOYBXTZYCZVVRCVURVWJVVEVURTFVVQVVDVURYBYAZYCZUUAWT
+      VVRVWAVWCUVSVVRVUEVUQVWFVWRYCVVRVUEVUSVWFVWTYCVWHVWMUUDXAUUEVVRVWEVVOJVVA
+      VVRVWBVVLVWDVVNXHVVRVWBBVVKIHZUVSQHVVLVVRVWAVXAUVSQVVRVUEBUWOVWFVWIVWQYDW
+      TVVRVVKCBVVRVUEUWOVWFVWQYCVWJVWIVWLVWKYEXAVVRVWDCVVMIHZUVSQHVXBVUCQHVVNVV
+      RVWCVXBUVSQVVRVUECVURVWFVWJVWSYDWTVVRUVSVUCVXBQUVOUVSVUCJVUGVVFVUDXJUUFVV
+      RVVMBCVVRVUEVURVWFVWSYCVWIVWJVWKVWLYEYFUUBOYFYLYGUUCVVGVVONFZVVAUVOVVFVUK
+      VXCUVOVVFGVUGVUJVXCUVOVUGVVFVUJVXCMVVQVVFVUJVXCVVQVVFVUJUUGVVLVVNVVQVVFVU
+      JVVLNFZVVRVUIVXDVUHVVRVUICVVKKLZVXDVVRUWKVUENFZVVDVUIVXEMUVNUWKUVMVUGVVFU
+      WLXKZVUGVXFUVOVVFVUEUKXIZVVQVVDVVEYHZCVUEUWOVTSVVRUWKCPWJZVVKNFVXEVXDRVXG
+      UVNVXJUVMVUGVVFUXIXKVVRVUEUWOVXHVXIYICVVKWKSYJUUHYKVVQVVFVUJVVNNFZVVRVUHV
+      XKVUIVVRVUHBVVMKLZVXKVVRUWHVXFVVEVUHVXLMUVMUWHUVNVUGVVFUWIYMZVXHVVQVVDVVE
+      UUIZBVUEVURVTSVVRUWHBPWJZVVMNFVXLVXKRVXMUVMVXOUVNVUGVVFUXHYMVVRVUEVURVXHV
+      XNYIBVVMWKSYJUUJYKUUKUULYNUUMYNOUUNVVGVUNVVJRZVVAVVGUWNNFZUWNPWJZVXFVXPUV
+      OVXQVUKVVFUVOUWNUYBUUOZXJUVOVXRVUKVVFUVOUWNUYBWLXJUVOVUGVVFVXFVUJVXHYGUWN
+      VUEWKSOUUPYLUUQYOYOWDVUNVUOVUPYPNFZUPVBNFUPUVDZVUNVUORYQUPYPNUURZVUNUDUPN
+      YRYSVXTVYAVUOVUPRYQVYBVUODUPNYRYSUUSVMZVUMVXQVUGVUNVULMUVOVXQVUKVXSOUVOVU
+      GVUJYHUWNVUEUVEVOWDUUTUVAUVBYTZUVOUVSUVPUVQVWGUVOUVPUVOUWNUVPEVYDUYBUVCWO
+      UYRUYSUVFWHYTUVOUWEUWFUWAUVOUWEGZUWFUVOUWAUWDYHVUMUVPVUEKLZMVYEUWFMUAAEVU
+      EAJZVUMVYEVYFUWFVYGVUKUWEUVOVYGVUGUWAVUJUWDVUEAEUVGVYGVUHUWBVUIUWCVUEABKX
+      BVUEACKXBXCXCUVHVUEAUVPKXBUVIVUMVUNVYFVYCUVOVUNVYFRVUKUVOUWNUVPVUEKVYDUVJ
+      OWHUVKUVLYLXL $.
+  $}
+
+  $( The product of two numbers' least common multiple and greatest common
+     divisor is the absolute value of the product of the two numbers.  In
+     particular, that absolute value _is_ the least common multiple of two
+     coprime numbers, for which ` ( M gcd N ) = 1 ` .
+
+     Multiple methods exist for proving this, and it is often proven either as
+     a consequence of the fundamental theorem of arithmetic or of
+     B&eacute;zout's identity ~ bezout ; see e.g.
+     ~ https://proofwiki.org/wiki/Product_of_GCD_and_LCM and
+     ~ https://math.stackexchange.com/a/470827 .  This proof uses the latter to
+     first confirm it for positive integer ` M ` and ` N ` (the "Second Proof"
```

Comment:
integers

Reply:
Sounds like a good change.

I guess the way Norm sometimes handled this was to say "in your next pull request, could you X?". So I've put this change (for both set.mm and iset.mm) on a branch but I guess I'll not put it into a pull request until I'd be making one anyway.

Reply:
Agreed. When I approve the PR while making some comments, the comments are only suggestions and/or can wait a future PR.

Reply:
Sounds good. If the branch is starting to get large, I'll definitely make a pull request, but it shouldn't cause trouble to combine this comment fix with the next little chunk of work.